Jupyter notebook support (#2357)
psf/black: docs/getting_started.md

_Black_ can be installed by running `pip install black`. It requires Python 3.6.2+ to
run, but can format Python 2 code too. Python 2 support needs the `typed_ast`

> dependency, which be installed with `pip install black[python2]`. If you want format
> Jupyter Notebooks as well, install with `pip install black[jupyter]`.

Reviewer: ```suggestion
dependency, which be installed with `pip install black[python2]`. If you want to format
Jupyter Notebooks, install with `pip install black[jupyter]`.
```
